modules/sdk/project-templates/project-templates-rest/src/main/resources/archetype-resources/build.gradle

```diff
@@ -17,9 +17,15 @@ dependencies {
 #if (${dependencyManagementEnabled} == "true")
 	compileOnly group: "javax.ws.rs", name: "javax.ws.rs-api"
 	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations"
+#if (${liferayVersion.startsWith("7.1")})
+	compileOnly group: "org.osgi", name: "org.osgi.service.jaxrs"
```

Comment:
We need to check to see if this artifact is in our compile.only BOMs.  If it is, then we can have this here, otherwise we should specify it explicitly for now.